sdk/python/kfp/cli/component.py

```diff
@@ -214,6 +215,13 @@ def _load_components(self):
             raise sys.exit(1)
         logging.info(f'Using target image: {self._target_image}')
 
+        pip_index_urls = []
+        for comp in self._components:
+            if comp.pip_index_urls is not None:
+                pip_index_urls.extend(comp.pip_index_urls)
+        if len(pip_index_urls) > 0:
```

Comment:
Per [Google Python styleguide 2.14](https://google.github.io/styleguide/pyguide.html#214-truefalse-evaluations), use implicit true for checking non-empty sequences

```suggestion
        if pip_index_urls:
```